doc/UserGuide.md

```diff
@@ -34,7 +34,7 @@ Format: `help`
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
 ### Adding a person: `add`
-Adds a person to the address book<br>
+Adds a person to the address book and list every record in the address book<br>
```

Comment:
Perhaps we can be more precise> Instead of `list every record` would `list all persons stored on the address book` be better?